Index: userPage/views.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/userPage/views.py b/userPage/views.py
--- a/userPage/views.py	(date 1697136096485)
+++ b/userPage/views.py	(date 1697136096485)
@@ -1,5 +1,6 @@
 from django.shortcuts import render,redirect
 from .forms import UserRegisterForm
+from django.views.decorators.csrf import csrf_exempt
 from django.http import HttpResponse
 from django.contrib.auth import authenticate, login, logout
 def register(request):
@@ -21,6 +22,6 @@
         password = request.POST["password"]
         user = authenticate(request,username=username,password=password)
         login(request,user)
-        return redirect("/")
+        return redirect("feed/main.html")
 
     return render(request,'userPage/SignIn.html')
